fv3core/utils/null_comm.py

```diff
@@ -0,0 +1,53 @@
+class NullAsyncResult:
+    def __init__(self, recvbuf=None):
+        self._recvbuf = recvbuf
+
+    def wait(self):
+        if self._recvbuf is not None:
+            self._recvbuf[:] = 0.0
+
+
+class NullComm:
+    """
+    A class with a subset of the mpi4py Comm API, but which
+    'receives' zeros instead of using MPI.
+    """
+
+    def __init__(self, rank, total_ranks):
+        self.rank = rank
+        self.total_ranks = total_ranks
```

Comment:
is there a reason not to keep with `size` as the default communication syntax for this?

Reply:
I used `total_ranks` to mirror `Partitioner.total_ranks`, which this is acting in place of. Normally the CubedSphereCommunicator doesn't have size, you get it from `cube.comm` instead.